database/migrations/2019_02_27_201348_increase_text_character_count_in_posts_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class IncreaseTextCharacterCountInPostsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('posts', function (Blueprint $table) {
+            DB::statement('ALTER TABLE posts MODIFY COLUMN text VARCHAR(500)');
```

Comment:
@chloealee Can we fix the `'NULL'` string default here using the `DB::statement()`, and then use the normal schema builder to change the column length? We don't want to be stuck making all our schema changes using manual SQL statements from now on!

Reply:
oh yes great call! Updated and when changing that, I could use the schema builder for both!